webform_civicrm.module

```diff
@@ -31,10 +31,12 @@ function webform_civicrm_library_info_alter(array &$libraries, $extension) {
     \Drupal::service('civicrm')->initialize();
     $user_framework_resource_url = CRM_Core_Config::singleton()->resourceBase;
     unset($libraries['civicrm_contact']['js']['/libraries/civicrm/packages/jquery/plugins/jquery.tokeninput.js']);
-    $libraries['civicrm_contact']['js'][$user_framework_resource_url . 'packages/jquery/plugins/jquery.tokeninput.js'] = [
-      'preprocess' => FALSE,
-      'minified' => FALSE
-    ];
+    if (substr($user_framework_resource_url, 0,-5) == 'core/') {
```

Comment:
Is there a typo here? Think we need to remove `0,` from the above to check if the last 5 char of the url matches `core/`

    substr($user_framework_resource_url, -5) === 'core/'

PHP 8 seems to have a straightforward fn for the same - https://www.php.net/manual/en/function.str-ends-with.php but seems we need to wait for that to use in code.

Also, should we have an else clause just in case the condition fails?

Reply:
Yes - typo! Fixed that.

Else -> not sure ... if we get there we'll have an unexpected `$user_framework_resource_url`